themes/default/monster/view.php

```diff
@@ -210,9 +210,9 @@
 		<td><?php echo htmlspecialchars($skill->INFO) ?></td>
 		<td><?php echo htmlspecialchars($skill->SKILL_LV) ?></td>
 		<td><?php echo htmlspecialchars(ucfirst($skill->STATE)) ?></td>
-		<td><?php echo $skill->RATE ?>%</td>
-		<td><?php echo $skill->CASTTIME ?>s</td>
-		<td><?php echo $skill->DELAY ?>s</td>
+		<td><?php echo $skill->RATE/100 ?>%</td>
```

Comment:
what will happen if skill->delay, for example, will be 0?
I mean this is a little bit dangerous operation inside.

Reply:
When the dividend (`$skill->RATE` in this case) is 0, the quotient is 0. It's when the divisor (100 in this case) is 0 that you are unable to complete the operation.

Reply:
nothing wrong dividing by 100 and ms to s by 1000